cfgov/jinja2/v1/index.html

```diff
@@ -197,6 +197,8 @@ <h2>Recent updates</h2>
                                 <h3 class="h4">
                                     {% if is_blog(post) %}
                                         {{ category_slug.render(category='blog', href=page_url) }}
+                                    {% elif is_event(post) %}
```

Comment:
This seemed like the least invasive way to make the change we wanted. I didn't want to update any category lists, since I wasn't sure how that would affect functionality elsewhere. But I'm open to other ways to accomplish this if anyone sees a nicer solution.

Reply:
I think this makes sense, given that `is_blog(post)` is already a pattern here.